actions/clean-up/dist/index.js

```diff
@@ -0,0 +1,3537 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
was it intentional to include this file?

Reply:
Yeah, this file is generated with the dependencies, so we don't need to check in node_modules.